playbooks/byo/openshift-etcd/migrate.yml

```diff
@@ -0,0 +1,78 @@
+---
+- include: ../openshift-cluster/initialize_groups.yml
+  tags:
+  - always
+
+- include: ../../common/openshift-cluster/evaluate_groups.yml
+  tags:
+  - always
+
+# TODO(jchaloup): replace the std_include with something minimal so the entire playbook is faster
+# e.g. I don't need to detect the OCP version, install deps, etc.
+- include: ../../common/openshift-cluster/std_include.yml
+  tags:
+  - always
+
+- name: Prepare masters for etcd data migration
+  hosts: oo_masters_to_config
+  tasks:
+  - set_fact:
+      master_services:
+      - "{{ openshift.common.service_type + '-master' }}"
+  - set_fact:
+      master_services:
+      - "{{ openshift.common.service_type + '-master-controllers' }}"
+      - "{{ openshift.common.service_type + '-master-api' }}"
+    when:
+    -  openshift.common.is_containerized
+  - set_fact:
+      master_services:
+      - "{{ openshift.common.service_type + '-master-controllers' }}"
+      - "{{ openshift.common.service_type + '-master-api' }}"
+    when:
+    - openshift.master.ha is defined and openshift.master.ha | bool and openshift_master_cluster_method == "native"
+    - not openshift.common.is_master_system_container | bool
```

Comment:
`openshift_master_cluster_method` is not always set so it could be:
```yaml
when:
    - (openshift_master_cluster_method is defined and openshift_master_cluster_method == "native") or openshift.common.is_master_system_container | bool
```

I was not sure about the condition. You are saying we want the condition for both `set_facts`? I.e. lines 42-54? Like merge both set_facts into one with the condition?